### PR TITLE
Replace Annotator with dist build

### PR DESCRIPTION
**Pull request by nickstenning**
Replaces inlined Annotator source with precompiled `.js` files for `Annotator` and the `Auth` and `Document` plugins.

### CODE REVIEW

**Comment by csillag** [comment]
As far as I can tell from reading the code, LGTM. 
(But I wonder what's the status of our extension builds. I don't remember seeing updates to those build scripts, so it might be broken.)

File: h/assets.yaml [hunk 1]
```diff
@@ -1,36 +1,25 @@
...
-      output: scripts/annotator/plugin/auth.js
-      contents: h:static/scripts/annotator/plugin/auth.coffee
+    - h:static/scripts/vendor/annotator.auth.js
     - filters: coffeescript
       output: scripts/annotator/plugin/bridge.js
       contents: h:static/scripts/annotator/plugin/bridge.coffee
-    - filters: coffeescript
-      output: scripts/annotator/plugin/document.js
-      contents: h:static/scripts/annotator/plugin/document.coffee
+    - h:static/scripts/vendor/annotator.document.js
```
Comment (gergely-ujvari) [inline h/assets.yaml]: Are we really using the document plugin in the sidebar? (If yes, for what?)

**Comment by gergely-ujvari** [comment]
Are these.js files upstream 1.2.x annotator files?

**Comment by csillag** [comment]
> Are these.js files upstream 1.2.x annotator files?

Yes, that's the point.

**Comment by csillag** [comment]
> (But I wonder what's the status of our extension builds. I don't remember seeing updates to those build scripts, so it might be broken.)

Yes, they are broken. (`icomoon.css` and `bootstrap.js` don't make it to the extension.)

**Comment by nickstenning** [comment]
> Yes, they are broken. (icomoon.css and bootstrap.js don't make it to the extension.)

In case it's not clear to others, this is not caused by this PR.

**Comment by tilgovi** [comment]
> Yes, they are broken. (icomoon.css and bootstrap.js don't make it to the extension.)

I think this may be only the case in development, and is not caused by this PR.

**Comment by gergely-ujvari** [comment]
LGTM

**Pull request merged**